2022/index.md

```diff
@@ -0,0 +1,42 @@
+## SMT-COMP 2022
+
+The *17th International Satisfiability Modulo Theories Competition (SMT-COMP
+2022)* is part of the 
+[SMT Workshop](https://smt-workshop.cs.uiowa.edu/) 2022, 
+affiliated with [IJCAR 2022](https://easychair.org/smart-program/IJCAR2022/).
+The SMT Workshop will include a block of time to
+present the competitors and results of the competition.
+
+### News
+{% assign newsList = site.news_2022 | reverse %}
+{% for news in newsList %}
+- {{ news.newsdate | date_to_string }} [{{ news.title }}]({{ news.url }})
+{% endfor %}
+
+### Key dates
+ - **May ??** &nbsp; Final versions of competition tools
+ - **June 17?**    &nbsp; Deadline for first versions of solvers (for all tracks)
+ - **July 8?**   &nbsp; Deadline for final versions of solvers, including system descriptions
```

Comment:
changed it.

I moved all dates to Saturday.  I think we had Sunday last year but that means that we can only really start running the Jobs on Tuesday, because the AOE deadline is on Monday noon.

Reply:
Sounds good to me